webrtc.html

```diff
@@ -2559,6 +2560,25 @@
             </ol>
           </dd>
 
+          <dt>RTCRtpTransceiver createRtpTransceiver(MediaStreamTrack or
+          DOMString track_or_kind, RTCRtpTransceiverInit init)</dt>
+
```

Comment:
The union type `MediaStreamTrack or DOMString` needs to be wrapped in parenthesis, i.e. `(MediaStreamTrack or DOMString) track_or_kind`